### DISCUSSION TITLE
ServerLoggerInterceptor  does not work in ServerHost project

### DISCUSSION THREAD

**Discussion by yaugkeer** [max-ieremenko/ServiceModel.Grpc#188] [General]
https://github.com/max-ieremenko/ServiceModel.Grpc/tree/master/Examples/grpc-dotnet-Interceptor/ServerSelfHost
ServerLoggerInterceptor works good in ServerAspNetHost, but  does not work in ServerHost project.
I just debug the code the link above, didn't change any code.
Anyone else can debug these code?
I guess that the configure has some bug:
 _server.Services.AddServiceModel<GreeterService>(
     serviceProvider,
     options =>
     {
         options.ConfigureServiceDefinition = definition =>
         {
             var interceptor = options.ServiceProvider!.GetRequiredService<ServerLoggerInterceptor>();
             return definition.Intercept(interceptor);
         };
     });

**Comment by max-ieremenko** [comment]
There seems to be a bug.
Thank you,

**Reply by max-ieremenko** [reply]
The bug is fixed in [version 1.8.1](https://github.com/max-ieremenko/ServiceModel.Grpc/releases/tag/1.8.1), please pull latest version.
